pkg/dispatcher/dispatcher_test.go

```diff
@@ -158,7 +58,55 @@ func TestFailToConsume(t *testing.T) {
 	}
 }
 
-func TestEndToEnd(t *testing.T) {
+func createRabbitAndQueue() (wabbit.Channel, *server.AMQPServer, error) {
+	fakeServer := server.NewServer(rabbitURL)
+	err := fakeServer.Start()
+	if err != nil {
+		return nil, nil, fmt.Errorf("failed to start RabbitMQ: %s", err)
+	}
+	vh := server.NewVHost("/")
+
+	ch := server.NewChannel(vh)
+	err = ch.ExchangeDeclare(exchangeName, "headers", // kind
+		wabbit.Option{
+			"durable":    false,
+			"autoDelete": false,
+			"internal":   false,
+			"noWait":     false,
+		},
+	)
+	if err != nil {
+		fakeServer.Stop()
+		return nil, nil, fmt.Errorf("failed to declare exchange: %s", err)
+	}
+
+	_, err = ch.QueueDeclare(queueName,
+		wabbit.Option{
+			"durable":    false,
+			"autoDelete": false,
+			"exclusive":  false,
+			"noWait":     false,
+		},
+	)
+
+	if err != nil {
+		ch.Close()
+		fakeServer.Stop()
+		return nil, nil, fmt.Errorf("failed to declare Queue: %s", err)
+	}
+
+	err = ch.QueueBind(queueName, "process.data", exchangeName, nil)
+
+	if err != nil {
+		ch.Close()
+		fakeServer.Stop()
+		return nil, nil, fmt.Errorf("failed to bind Queue: %s", err)
+	}
+	return ch, fakeServer, nil
+}
+
+/* func TestEndToEnd(t *testing.T) {
+	t.Skip()
```

Comment:
can we uncomment the block and leave the t.Skip()? And maybe add a dedicated comment why we're briefly skipping these tests